pkgs/data/themes/juno/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchurl, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  pname = "juno";
+  version = "0.0.1";
+
+  srcs = [
+    (fetchurl {
+      url =
+        "https://github.com/gvolpe/Juno/releases/download/${version}/Juno.tar.xz";
+      sha256 = "1cghsn9qagcf1nlga5cal0aqch6hkjm5wk6ja791zxhdqy3crx1i";
+    })
+    (fetchurl {
+      url =
+        "https://github.com/gvolpe/Juno/releases/download/${version}/Juno-mirage.tar.xz";
+      sha256 = "0zh6bc85svmwh8qrhpn8mim0pj322x2x2i9sxnp7p1938p5z5m2b";
+    })
+    (fetchurl {
+      url =
+        "https://github.com/gvolpe/Juno/releases/download/${version}/Juno-ocean.tar.xz";
+      sha256 = "0m2wgmcn12dfq5badzlpzjc8792ba9hi32c79vfvqawdn1q3hrdx";
+    })
+    (fetchurl {
+      url =
+        "https://github.com/gvolpe/Juno/releases/download/${version}/Juno-palenight.tar.xz";
+      sha256 = "1hn2l0m76x61ixjd253hi7czm65asdjdhqvvlv7idbccc40pvrak";
+    })
+  ];
+
+  sourceRoot = ".";
+
+  propagatedUserEnvPkgs = [ gtk-engine-murrine ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/themes
+    cp -a Juno* $out/share/themes
+    rm $out/share/themes/*/{LICENSE,README.md}
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "GTK themes inspired by epic vscode themes";
+    homepage = "https://github.com/EliverLara/Juno";
+    license = licenses.gpl3;
```

Comment:
gpl3Only or gpl3Plus?

Reply:
https://github.com/EliverLara/Juno/blob/master/LICENSE

Reply:
I would say only because later is not mentioned anywhere.

Reply:
I don't know what you mean, should this be changed?

Reply:
In these cases, it is a good idea to open an issue upstream for clarification. See also https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
I still don't know which one I should use, can anyone clarify?

Reply:
`gpl3Only` is the safe choice but it is possible upstream meant `gpl3Plus` so it would be best to ask them ([example](https://github.com/rime/ibus-rime/issues/107)).

Reply:
Thanks, I'll ask upstream :+1:

Reply:
Updated to `gpl3Only`. I [raised an issue upstream](https://github.com/EliverLara/Juno/issues/26) but I doubt I'll get a response anytime soon, most of the issues don't and the last response from the author was in June 2020. I think we should merge and I'll take care of updating it (if needed) if I ever get a response.